### PR TITLE
Added Reusable Partials for Inky Templates

### PR DESCRIPTION
**Pull request by philson-philip**
@philson-philip _a Added the reusable partials wrt neetoAuth project. Please check it and share your views.
cc: @unnitallman @karthiknmenon

### CODE REVIEW

**Comment by karthiknmenon** [comment]
@goutham-subramanyam _a Could you please verify these changes? 

cc: @philson-philip

File: app/assets/stylesheets/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,146 @@
+//  Foundation for Emails Settings
```
Comment (goutham-subramanyam) [inline app/assets/stylesheets/_variables.scss:1]: Most of the variables here aren't being used. Please clean up the file.
Also follow neetoui variable naming.

File: app/assets/stylesheets/template/_template.scss [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+  padding-top: 15px;
+}
+
+.header .wrapper-inner {
+  padding: 20px;
+}
+
+.header .container {
+  background: transparent;
+}
```
Comment (goutham-subramanyam) [inline app/assets/stylesheets/template/_template.scss:40]: Move these styles to layouts.scss or overrides.scss

File: app/assets/stylesheets/template/_template.scss [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+  .wrapper-inner {
+    padding: 0 64px;
+
+    @media only screen and (max-width: 768px) {
+      padding: 0 24px;
+    }
+  }
+}
+
+.neeto-emails-title {
```
Comment (goutham-subramanyam) [inline app/assets/stylesheets/template/_template.scss:52]: Do these classes get converted to inline styles in the output? 
If yes, clean up the code and use variables for font size, weight, etc.
If it doesn't get converted, then move these styles to be inline.

File: app/views/neeto_emails/layouts/_footer_preferences.html.inky [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<spacer size="16"></spacer>
+<div class="neeto-emails-text--footer">
+  <p>Update your <a href="https://neeto.com">email preferences</a> to choose the types of emails you receive, or you can <a href="https://neeto.com">unsubscribe </a>from all future emails.</p>
```
Comment (goutham-subramanyam) [inline app/views/neeto_emails/layouts/_footer_preferences.html.inky:3]: The links should be customizable right?

File: app/views/neeto_emails/layouts/_greeting_section.html.inky [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<div class="neeto-emails-text--thank-you">
+  <p>
+    Thank You<br />
+    <span>The neeto team </span>
```
Comment (goutham-subramanyam) [inline app/views/neeto_emails/layouts/_greeting_section.html.inky:4]:
```suggestion
    <span>The neeto team</span>
```

File: app/views/neeto_emails/layouts/_logo_section.html.inky [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<img
+  alt="neeto"
+  height="auto"
+  style="height: 34px; width: auto"
```
Comment (goutham-subramanyam) [inline app/views/neeto_emails/layouts/_logo_section.html.inky:4]:
```suggestion
  style="height: 32px; width: auto"
```

File: app/views/neeto_emails/layouts/_primary_link_section.html.inky [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<div class="neeto-emails-btn neeto-emails-btn--primary">
+  <a href="<%= link %>" target="_blank">
+    <%= label %>
+  </a>
+</div>
```
Comment (goutham-subramanyam) [inline app/views/neeto_emails/layouts/_primary_link_section.html.inky:5]: Do we need a div wrapper for the button?

File: app/views/neeto_emails/layouts/_title_section.html.inky [hunk 8]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (goutham-subramanyam) [inline app/views/neeto_emails/layouts/_title_section.html.inky:1]:
```suggestion
```

File: app/views/neeto_emails/layouts/common.html.erb [hunk 9]
```diff
@@ -4,6 +4,7 @@
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta name="viewport" content="width=device-width" />
 
+    <link href="https://api.fontshare.com/css?f[]=satoshi@300,400,500,600,700,900&display=swap" rel="stylesheet">
```
Comment (goutham-subramanyam) [inline app/views/neeto_emails/layouts/common.html.erb:7]: I don't think we can use web fonts in email templates. Please refer: https://jsfiddle.net/wallyglenn/g35nchmL/

**Review by goutham-subramanyam — DISMISSED** [review]
@philson-philip _a

File: app/views/neeto_emails/layouts/_footer_preferences.html.inky (hunk 4 above)
Comment (philson-philip) [reply app/views/neeto_emails/layouts/_footer_preferences.html.inky:3]: @unnitallman _a Can you please check it?

File: app/views/neeto_emails/components/_footer_preferences.html.inky [hunk 10]
```diff
@@ -0,0 +1,4 @@
+<spacer size="16"></spacer>
+<div class="neeto-emails-text--footer">
+  <p>Update your <a href="https://neeto.com">email preferences</a> to choose the types of emails you receive, or you can <a href="https://neeto.com">unsubscribe </a>from all future emails.</p>
```
Comment (unnitallman) [inline app/views/neeto_emails/components/_footer_preferences.html.inky:3]: @philson-philip Please make the links customizable, as per the update we had on Slack channel. Let me know once done, will merge this PR.

Comment (philson-philip) [reply app/views/neeto_emails/components/_footer_preferences.html.inky:3]: @unnitallman _a Can you take over this?

**Pull request merged**